framework/_partials/_preview-datepickers.twig

```diff
@@ -0,0 +1,34 @@
+{% set bodyClass = '' %}
+{% if _target.context is defined and _target.context.global is defined %}
+  {% if _target.context.global.svg is defined %}
+    {% set bodyClass = 'no-svg' %}
+  {% endif %}
+{% endif %}
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="{{ '/framework/styles/europa.css' | path }}">
+
+    <title>Preview Layout for datepickers</title>
+  </head>
+  <body{% if _target.context and _target.context.global and _target.context.global.language %} class="language-{{ _target.context.global.language }}"{% endif %}>
+    {{ yield }}
+    <script src="{{ '/framework/scripts/europa.js' | path }}"></script>
+    {% if _target.context._demo and _target.context._demo %}
+    <script type="text/javascript">{{ _target.context._demo.scripts | raw }}</script>
+    {% endif %}
+
+    <!-- Specific scripts included in this preview template -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/pikaday/1.6.1/pikaday.min.js"></script>
```

Comment:
A bit messy but it works. Maybe by using proper template inheritance we could avoid rewriting the same code over and over again